cvat-ui/src/components/create-task-page/create-task-content.tsx

```diff
@@ -442,6 +452,21 @@ class CreateTaskContent extends React.PureComponent<Props & RouteComponentProps,
             return;
         }
 
+        try {
+            this.validateQualityParams(quality);
+        } catch (e) {
+            if (e instanceof Error) {
+                notification.error({
+                    message: 'Could not create a task',
+                    description: e.message,
+                    className: 'cvat-notification-create-task-fail',
+                });
+                reject();
+                return;
+            }
+            throw e;
```

Comment:
Why do you throw the error outside of the async function? There is not such statement in other checks